include/common/graphic-helpers.h

```diff
@@ -9,29 +9,49 @@ struct wlr_scene_tree;
 struct wlr_scene_rect;
 struct wlr_fbox;
 
+struct border_rect {
```

Comment:
Toolkits and libraries normally just call "rectangles with borders" rects. In other words there is an assumptions that when you draw a rectangle, you can specify borders. I suppose the `wlr_scene_rect` is quite a basic object in that sense. For a more intuitive API, I'd suggest dropping the `border_` prefix.

So, suggest considering calling it `lab_scene_rect` or simply `rect`